types/common_label.go

```diff
@@ -16,6 +16,24 @@ const (
 	LabelFrontendEntryPoints                     = LabelPrefix + "frontend.entryPoints"
 	LabelFrontendRequestHeader                   = LabelPrefix + "frontend.headers.customrequestheaders"
```

Comment:
if camelcase then these should be updated as well

Reply:
Please open an issue instead of commented an already merged PR.

Reply:
@vito-c or open a PR. thanks